client/src/app/users/add-user.component.html

```diff
@@ -23,7 +23,7 @@
 
           <mat-form-field>
             <mat-label>Age</mat-label>
-            <input matInput #input id="ageField" type="number" placeholder="Age"
+            <input matInput #input id="ageField" placeholder="Age"
```

Comment:
I think it's pretty important to present a numerical input to the user. Can you elaborate on the issue you were seeing with this field having type "number"?

Reply:
It probably makes sense to at least use `inputmode="numeric"` on the input like we do in Buzz About.

Reply:
I would still prefer a real number input if possible because the browser's UI for it makes a lot of sense here.

Reply:
That makes sense. If you use `type="number"` and then the user enters a non-numeric input, the form field will behave as though the user hasn't entered any value at all—it'll say "Age is required" instead of "Age must be a number".

The alternative, which I'd also be fine with, is to rely on the validation inherent to `input type="number"`, and not in add our own validation. 

(We'd have to do a bit of fiddling with the tests so that they understand the validation inherent to `input type="number"`, though—the fact that we'd have to refactor the tests a bit is part of the reason I avoided going that route.)

Reply:
Oh, also, the inherent validation messages are not as nice as the ones we can provide ourselves.

Reply:
I think the fact the user simply can't enter non-numerical values in a number field is OK. Does the rest of the validation work, eg if a user enters 0 or -20?

Reply:
Yes, the rest of the validation works just fine.

Thanks for your feedback! I'll try to change that over to using the built-in `type="number"` validation.